test/sharness/t0021-config.sh

```diff
@@ -111,10 +111,10 @@ test_config_cmd() {
 
   test_expect_success "'ipfs config replace' injects privkey back" '
        ipfs config replace show_config &&
-	   grep "\"PrivKey\":" "$IPFS_PATH/config" | grep -e ": \".\+\"" >/dev/null
+       grep "\"PrivKey\":" "$IPFS_PATH/config" | grep -e ": \".\+\"" >/dev/null
```

Comment:
hard time reconciling this test case with https://github.com/ipfs/go-ipfs/pull/3141/files#r77544560. I thought it would error out on this case (which i would consider a bug-- i.e. i agree with this test case, it should not error to provide a config without key).